### ISSUE TITLE
Double-layered comments?

### ISSUE THREAD

**Issue by JerryNixon** [RealDotNetDave/dotNetTips.Spargine#64]
Don't you have Source Control so you don't need to do this?

https://github.com/RealDotNetDave/dotNetTips.Spargine/blob/6883de33184f36f36e430aa479e670023d011cc7/source/6/dotNetTips.Spargine.6.Core/Logging/LoggingHelper.cs#L33-L61

**Comment by RealDotNetDave** [comment]
Yes. I've been waiting for .NET 7 to come out before I remove it.
